01_P/P_2_1_2_01/sketch.js

```diff
@@ -44,9 +44,7 @@ function draw() {
   translate(width/tileCount/2, height/tileCount/2);
 
   background(255);
-  smooth();
```

Comment:
Removed smooth() since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3). Moved `noFill` into `setup` because it only has to be set once.